static/jsfiles/view1.js

```diff
@@ -134,28 +121,27 @@ function fill_metadata(csvContent) {
         lstItem.appendChild(svgElement);
 
         const nameText = document.createTextNode(data2[key]['name']);
+        console.log("nameText : " + nameText)
```

Comment:
Remove here as well

Reply:
There are 17 occurrences of console.log(), should I remove them all ?

Reply:
Can we create/use some sort of logger to change the verbosity between dev and deployment flows? And this can be wrapped with the logger.

Reply:
done